pages/index.js

```diff
@@ -1,7 +1,10 @@
-import Image from "next/image";
-import Link from "next/link";
-import Layout, { siteTitle } from "../components/layout";
+import PageDetails from "../components/organisms/PageDetails";
+import Layout from "../components/layout";
 
 export default function Home() {
-  return <Layout home></Layout>;
+  return (
+    <Layout home>
+      <PageDetails />
```

Comment:
Wouldn't this be better to have in the layout above the footer?

Reply:
I agree!